src/Board.elm

```diff
@@ -0,0 +1,25 @@
+module Board exposing (..)
+
+import List exposing (repeat, take, drop, head)
+
+type alias Board = List String
```

Comment:
Yeah, I had forgotten about Array! I think having had trouble with Array in Java, and List being the answer, I was assuming the same here. Theres also a chance a Dict could work well. Not sure yet.